src/Site_WP_Docker.php

```diff
@@ -142,6 +142,28 @@ public function generate_docker_compose_yml( array $filters = [] ) {
 		];
 		$mailhog['networks']     = $network_default;
 
+		// postfix configuration.
+		$postfix['service_name'] = array( 'name' => 'postfix' );
+		$postfix['image']      = array( 'name' => 'easyengine/postfix:v' . EE_VERSION );
```

Comment:
The entire file is using new array notation `[]`, please try and use that to maintain code consistency and run phpcbf to update formatting.